config/jobs/kubernetes/sig-autoscaling/sig-autoscaling-presubmits.yaml

```diff
@@ -89,6 +89,12 @@ presubmits:
     decoration_config:
       timeout: 310m
     run_if_changed: '^(pkg\/controller\/podautoscaler\/|test\/e2e\/autoscaling\/custom_metrics_stackdriver_autoscaling.go$)'
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: release
```

Comment:
Oh I meant k/k but this is a presubmit so it has it.
You shouldn't need to clone k/release you just need the top level path alias for k/k.